### PR TITLE
this is a sketchup. davvy dont insult me

### PR DESCRIPTION
**Pull request by zrodevkaan**

### CODE REVIEW

**Comment by zrodevkaan** [comment]
Let me know what to change and add. I am terrible at explaining

File: README.md [hunk 1]
```diff
@@ -1,2 +1,86 @@
...
+
+// Replugged
+replugged.webpack.getByProps('sizeLarge').sizeLarge
+
+// Vencord
+findByPropsLazy('sizeLarge').sizeLarge
+```
+
+How AutoCSS works is like this, you pass in some props and it works, but 
+sometimes youll have to add more props to find your className
```
Comment (Davilarek) [inline README.md]: missing `'` after "you"

Comment (zrodevkaan) [reply README.md]: i hate you

**Comment by Davilarek** [comment]
why does this pr point to fetchMethod branch btw?

**Comment by zrodevkaan** [comment]
We are going to merge fetchMethod into main

**Comment by zrodevkaan** [comment]
i pressed the wrong button

**Comment by zrodevkaan** [comment]
@Davilarek review this please. now

with a cherry on top

**Comment by Davilarek** [comment]
looks fine to me

**Comment by zrodevkaan** [comment]
davvy-chan, thankies you ;3 uwu

**Pull request merged**